### DISCUSSION TITLE
ECU communication while engine off

### DISCUSSION THREAD

**Discussion by cpat50** [cedricp/ddt4all#842] [Q&A]
Hello,
I'm trying to activate lane keeping assist in the electrical power steering section. There is a warning message saying the engine has to be turned off to proceed to the activation.
The thing is, if I turn it off the ECU is not communicating anymore (at least with ddt4all).
So what's the trick ?
Thank you

**Comment by AndyIreland** [comment]
Hi, when they say 'engine off' it more than likely means engine turned off , but car in standby / accessories mode .
so - depending what renault you have. The one with a flat key card its called 'parking mode'
Put the keycard into the slot in the dash and put the gear into D (Drive) and (without starting the engine - ie dont press on any pedals) long press the 'start' button on the dash until 'please remove keycard' appears on dash (but leave the keycard in the slot) 

If you have a manual car, (ie not automatic transmission):
Put the keycard into the slot in the dash and put the gear into 1st gear and (without starting the engine - ie dont press on any pedals) long press the 'start' button on the dash until 'please remove keycard' appears on dash (but leave the keycard in the slot) 

If your renault has a key fob instead (later renaults) 
leave your key fob on the centre console instead and select 1st gear (or Drive) and press start button for a few seconds until something like 'ACC' comes up on the dashboard cluster

**Reply by cpat50** [reply]
Thank you Andy it worked with the last suggestion (2016 Talisman) !
I may be wrong but I couldn't find those procedures anywhere. 
@cedricp it could be useful to talk about it somewhere in your guidelines/FAQ
